config/dask/snellius/jobqueue.yaml

```
@@ -0,0 +1,46 @@
+jobqueue:
+  slurm:
+    name: dask-worker
+
+    # Dask worker options
+    cores: 4                  # Total number of cores per job
```

Maybe we can set here 32 as default number of cores per job? Anyway users are assigned this amount of cores per job on Snellius right?
```suggestion
    cores: 32                 # Total number of cores per job
```
and not sure about the memory available on a thin node for 32 cores?